Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = [
+  # "crates/admin",
```

Comment:
What is this admin crate that keeps popping up as a comment?

Reply:
It's the holochain_admin crate I've made to launch / configure conductors. I'd like to pull it out of the workspace but need the interface types to do so. So probably can now.